ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/panelcontainer/v1/panelcontainer/_cq_dialog/.content.xml

```diff
@@ -217,23 +217,23 @@
                                     <breakBefore
                                             jcr:primaryType="nt:unstructured"
                                             sling:resourceType="fd/af/granite/components/paginationpicker"
-                                            fieldLabel="Place"
+                                            fieldLabel="Location of the panel in Document of Record"
```

Comment:
Pagination is more aligned to XFA spec - and the new strings are confusing for existing users. Needs review.